### ISSUE TITLE
Cake.Wyam not referencing latest Cake.Core

### ISSUE THREAD

**Issue by Jericho** [statiqdev/Statiq#701]
Cake.Wyam references Cake.Core 0.26.0 resulting in the following message: 

> The assembly 'Cake.Wyam, Version=1.5.0.0, Culture=neutral, PublicKeyToken=null'
> is referencing an older version of Cake.Core (0.26.0).
> For best compatibility it should target Cake.Core version 0.28.0.

**Comment by daveaglick** [comment]
I left it at 0.26.0 on purpose - that's the minimum version required for >= 0.28.0 compatibility, even with the warning. It's a very simple addin, so it doesn't hit any of the potentially breaking changes is 0.28.0 and up.

I could be talked into updating the Cake dependency, but that might exclude anyone using Cake 0.26.0 through 0.27.2 and really the only benefit would be getting rid of that warning.

Thoughts?

**Comment by Jericho** [comment]
I am the self-appointed Cake addins guidelines enforcer and you, sir, are in violation of the [Supported Cake Version](https://github.com/cake-contrib/Home/blob/master/Guidelines.md#supported-cake-version) guideline which causes red marks next to your addin in the Cake 0.28.0 [audit](https://github.com/cake-contrib/Home/blob/master/Audit.md) :stuck_out_tongue_winking_eye:

Joking aside, I am told that addins targeting 0.28.0 should also be compatible with 0.26.0 as long as they don't rely on new features/breaking changes, of course. Therefore upgrading your reference to 0.28.0 should make your addin compatible with both 0.26 and 0.28 AND eliminate the warning.

**Comment by daveaglick** [comment]
> you, sir, are in violation of the Supported Cake Version guideline

😱

That’s what I get for drinking and coding. Guess I’ll have to do some community service and get that version bumped.

**Comment by Jericho** [comment]
I'm just curious: is there a reason why you didn't create a separate repo for the addin? I'm asking because I struggled a little bit to find where the source for the addin was and where to open this issue. It seems to me that it would be easier if this addin was in it's separate repo (just my opinion though). And also it could allow you to transfer the addin repo to the cake-contrib organization (if you so desire) without affecting Wyam's repo.

**Comment by daveaglick** [comment]
I sync-ship the Cake addin on releases so it’s easier to have a single build and deployment script take care of everything. It also helps remind me to update the Cake addin when I adjust the CLI.

That said, I’ve actually been thinking about splitting it out (along with other stuff like the themes) during the major refactoring I’m about to start for 2.0. There’s probably something to be said for separating the versioning and release schedule of the Cake addin as well.

TL;DR: I’m lazy and this was easier at the time :smile:

**Comment by daveaglick** [comment]
Bumped the Cake ref and new version going out now
